### DISCUSSION TITLE
Report: Categories - Summary

### DISCUSSION THREAD

**Discussion by TomAff** [moneymanagerex/moneymanagerex#5393] [Q&A] [answered]
can I get the grm file of this Report?
Is it possible to show the Month of the current year in separate columns?

**Comment by tactilis** [comment, marked as answer]
This report is built-in and therefore doesn't have a grm.

**Reply by TomAff** [reply]
So it is not possible to get the html-template in any way?

**Reply by tactilis** [reply]
The reports are built in the C++ code.

See for example: https://github.com/moneymanagerex/moneymanagerex/blob/master/src/reports/categexp.cpp

Line 312 and 106 to 298

**Reply by TomAff** [reply]
Ok. Thanks
